autoPyTorch/api/base_task.py

```diff
@@ -600,6 +602,14 @@ def _do_traditional_prediction(self, num_run: int, time_for_traditional: int) ->
             status, cost, runtime, additional_info = future.result()
```

Comment:
Can you encapsulate this in a try/except block. In dask, when you get the result of a configuration, if an exception happen, it will be raised when collecting the result. If 1 configuration failed, we can show the error to the user but 'continue' as other configurations maybe work.

Reply:
As far as I understand, if we have a failed configuration, getting results from dask will work fine. By 'if an exception happen' do you mean an exception that happens outside of the TAE block?